docs/using_edge_services/service_rollbacks.md

```diff
@@ -131,4 +131,4 @@ hzn eventlog list
 ```
 {: codeblock}
 
-Lastly, you can also use the [management console](../console/accessing_ui.md) to modify rollback deployment versions settings. You can do this when creating a new deployment policy or by viewing and editing existing policy details including rollbacks settings. Note the term “time frame” in the rollback section of the UI is equivalent to “retry_durations” in the CLI.
+Lastly, you can also use the [management console](../nl/es/console/accessing_ui.md) to modify rollback deployment versions settings. You can do this when creating a new deployment policy or by viewing and editing existing policy details including rollbacks settings. Note the term “time frame” in the rollback section of the UI is equivalent to “retry_durations” in the CLI.
```

Comment:
Open Horizon does not have a UI.  We should remove the whole "Lastly" paragraph as it does not apply.

Reply:
ok